tests/unit/helpers.js

```diff
@@ -378,11 +392,13 @@ export const createSecondaryHeroShelf = ({
   };
 };
 
-export const createHeroShelves = ({
+export const createHomeShelves = ({
+  resultsProps,
```

Comment:
could we set a default value to this param instead of having a fallback later? Maybe something like this?

```suggestion
  resultsProps = [fakeShelf],
```